_config/graphql.yml

```diff
@@ -15,3 +15,7 @@ SilverStripe\GraphQL\Schema\Schema:
     admin:
       src:
         - 'silverstripe/cms: _graphql'
+SilverStripe\Core\Injector\Injector:
```

Comment:
I've already added this on the original location (a bit further up) in the same file: https://github.com/silverstripe/silverstripe-cms/blob/4/_config/graphql.yml#L6-L9